CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+1.17.0 (2023-09-11)
+-------------------
```

Comment:
It seems this line is added at release time usually (visible on this [release PR](https://github.com/adamchainz/blacken-docs/commit/960ead214cd1184149d366c6d27ca6c369ce46b6))
```suggestion
```